### ISSUE TITLE
Add possibility add custom links to admin list block in dashboard

### ISSUE THREAD

**Issue by Koc** [sonata-project/SonataAdminBundle#2953]
We have some actions, like simple list. And there is requirement add link to simple list near create and list actions in block

**Comment by rccc** [comment]
Hello,

 Are you talking about this one : https://sonata-project.org/bundles/admin/master/doc/reference/dashboard.html#the-admin-list-block ?

**Comment by Koc** [comment]
Yes

**Comment by rccc** [comment]
Hello,

Maybe you cas try with the 'list_box' option :
https://sonata-project.org/bundles/admin/master/doc/reference/templates.html#other-templates
https://sonata-project.org/bundles/admin/master/doc/reference/dashboard.html#blocks

++

**Comment by Koc** [comment]
Of course, we have created custom template for `list_block` but it looks so ugly: too much code duplicates. Better add some option for template which can be included in this list, like

```twig
{% if admin.hasTemplate('additional_items') %}
    {{ include(admin.getTemplate('additional_items')) }}
{% endif %}
```

**Comment by Koc** [comment]
Closed by #3483
